code/game/objects/items/dragon.dm

```diff
@@ -0,0 +1,56 @@
+//Coded by Nexendia~, ported by Spacedong
+
+/obj/item/dragon
+	name = "Bad Dragon"
+	desc = "You really shouldn't see this.. but if you do... Huzzah.. you found a bug.. please scream at Spacedong!"
+	icon = 'icons/obj/dicks.dmi'
+	icon_state = null
+	force = 5
+	w_class = 1
+	throwforce = 5
+	hitsound = 'sound/items/squishy.ogg'
+	attack_verb = list("slapped")
+
+/obj/item/dragon/suicide_act(mob/user)
+	user.visible_message(pick("<span class='suicide'>[user] is shoving [src.name] down \his throat! It looks like they're trying to commit suicide.</span>"))
```

Comment:
did you just

assume

their gender


also we have 3 genders in our code git it rite